### DISCUSSION TITLE
Opposite direction shown by stochastic vs dynamical mode

### DISCUSSION THREAD

**Discussion by mayurdoke6** [theislab/scvelo#947] [Q&A]
Hello  @VolkerBergen and @stefanpeidli! Thank you for this wonderful tool Scvelo. I have a question similar to issues  #112 and #456. I have tried all suggestions made by contributors made in the comments but it did not change my outcome. So, I have a basic question in the case of the reliability of using either stochastic or dynamical mode for the publication. As you can see in the attached figure the stochastic mode arrows make more sense biologically (especially the green color cluster) than the dynamical mode. The pseudotime and latent time graphs which were generated using stochastic and dynamical modes respectively identify similar root and endpoints. Interestingly, the PAGA graph arrows generated from both modes showed 90% similarity. 
However, in other RNA velocity analysis UMAPs, we found that the dynamical mode showed better biological relevant results than the stochastic mode. 
So my question is how we can justify these discrepancies in results to reviewers. Can we use the best-suited model for each different analysis and use it for publication? Please let me know your view on this @WeilerP 
![image](https://user-images.githubusercontent.com/43390895/196798217-1dd597a7-3d83-4d8e-aa6d-e7a3b1de8d2e.png)

**Comment by WeilerP** [comment]
@mayurdoke6, please have a look at e.g. https://github.com/theislab/scvelo/issues/216 or https://github.com/theislab/scvelo/discussions/462.
